src/styles/components/application-loading-indicator/styles.less

```diff
@@ -1,25 +1,39 @@
-@keyframes application-loading-indicator-pulse {
-  0% {
-    opacity: 0.4;
-  }
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+application-loading-indicator/styles.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
 
-  50% {
-    opacity: 0.15;
-  }
 
-  100% {
-    opacity: 0.4;
+
+& when (@application-loading-indicator-enabled) {
+
+  @keyframes application-loading-indicator-pulse {
```

Comment:
If you want to be consistent with all styling in CNVS, we'd have extra space after opening and before closing `{ }`.  Not a blocker, but helps with consistency.